stories/adminbar.stories.js

```diff
@@ -73,6 +74,8 @@ storiesOf( 'Global', module )
 			'googlesitekit.SearchConsole',
 			addGoogleSitekitSearchConsoleAdminbarWidget );
 
+		disableFeature( 'widgets.adminBar' );
```

Comment:
As per my comment above, this should be automatically disabled for every story by default, and only enabled where relevant.